main.py

```diff
@@ -1,10 +1,37 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+    for name, data in data.items():
+        race, _ = Race.objects.get_or_create(
+            name=data.get("race").get("name"),
+            description=data.get("race").get("description")
+        )
+        guild = data.get("guild")
+        if guild:
+            guild, _ = Guild.objects.get_or_create(
+                name=data.get("guild").get("name"),
+                description=data.get("guild").get("description")
+            )
+        Player.objects.get_or_create(
+            nickname=name,
+            email=data.get("email"),
+            bio=data.get("bio"),
+            race=race,
+            guild=guild
```

Comment:
The `guild` variable is initially set to `None` if the `guild` key is not present in the data. However, when `guild` is not present, the `Player.objects.get_or_create` call will attempt to use `None` as the `guild` value. Ensure that the `guild` field in the `Player` model allows `null=True` to avoid potential errors.